ansible/roles/test/files/ptftests/vnet_vxlan.py

```diff
@@ -300,11 +301,23 @@ def setUp(self):
         self.generate_ArpResponderConfig()
 
         self.cmd(["supervisorctl", "start", "arp_responder"])
-
+        self.check_arp_responder_running()
         self.dataplane.flush()
 
         return
 
+    def check_arp_responder_running(self):
+        """
+        Check arp_responder is in RUNNING state, if not sleep 1 sec and check again
+        """
+        for i in range(5):
+            output = self.cmd(["supervisorctl", "status", "arp_responder"])
+            if 'RUNNING' in output[0]:
+                break
+            time.sleep(1)
+        else:
+            raise Exception("arp_responder state is not RUNNING! Output: %s" % output)
```

Comment:
Why not using wait_until?

Reply:
Hi @yxieca 

This is executed inside the  ptf docker, it does not has sonic_mgmt to do imports .
I did check the https://github.com/p4lang/ptf/ and did not find suitable